src/legacy/server/config/schema.js

```diff
@@ -135,7 +135,14 @@ export default () => Joi.object({
         then: Joi.default(!process.stdout.isTTY),
         otherwise: Joi.default(true)
       }),
-    timezone: Joi.string()
+    timezone: Joi.string(),
+    rotate: Joi.object().keys({
+      enable: Joi.boolean().default(false),
```

Comment:
Let's change this to `enabled` to keep consistent with the tense we use elsewhere.

Reply:
👍